packages/table/src/util/downloadTable.js

```diff
@@ -74,8 +73,28 @@ export function downloadJson(tableData, table, downloadFileName) {
   let tempLink = '';
   tempLink = document.createElement('a');
   tempLink.setAttribute('href', JsonURL);
-  tempLink.setAttribute('download', createFileName(downloadFileName || ''));
+  tempLink.setAttribute('download', createFileName(downloadFileName || '', 'csv'));
   document.body.appendChild(tempLink);
   tempLink.click();
   document.body.removeChild(tempLink);
 }
+
+export function downloadJson(tableData, table, downloadFileName) {
```

Comment:
@Senait7-ux , Would it be good idea to have single function to handle generation of CSV or JSON? The reasoning behind it is that it looks like both functions have kind of similar code.

Reply:
@iksheth , yes it would be a good idea since both of them use the same download mechanism